src/patternfly/components/VerticalNav/styles.scss

```diff
@@ -0,0 +1,134 @@
+  --pf-c-vertical-nav--BackgroundColor:   var(--pf-global--BackgroundColor--light-100);
+
+  // Vertical nav link
+  --pf-c-vertical-nav__link--PaddingTop:      var(--pf-global--spacer--lg);
+  --pf-c-vertical-nav__link--PaddingRight:    calc(var(--pf-global--spacer--lg) + .875rem + var(--pf-c-vertical-nav__link__arrow--Right));
+  --pf-c-vertical-nav__link--PaddingBottom:   var(--pf-global--spacer--lg);
+  --pf-c-vertical-nav__link--PaddingLeft:     var(--pf-global--spacer--xl);
+  --pf-c-vertical-nav__link--MarginTop:       0;
+  --pf-c-vertical-nav__link--MarginBottom:    0;
+  
+  // Vertical nav link active :after
+  --pf-c-vertical-nav__link--after--active--Left:    calc(var(--pf-c-vertical-nav__link--PaddingLeft) + 1.5rem + var(--pf-c-vertical-nav__link__icon--MarginRight));
+  --pf-c-vertical-nav__link--after--active--Width:   2rem;
+  --pf-c-vertical-nav__link--after--active--Height:  .25rem;
+  --pf-c-vertical-nav__link--after--active--Color:   var(--pf-global--Color--active);
+
+  // Vertical nav link colors
+  --pf-c-vertical-nav__link--Color:             var(--pf-global--Color--dark-100);
+  --pf-c-vertical-nav__link--hover--Color:      var(--pf-global--Color--dark-100);
+  --pf-c-vertical-nav__link--active--Color:     var(--pf-global--Color--active);
+  --pf-c-vertical-nav__link--focus--Color:      var(--pf-global--Color--dark-100);
+  --pf-c-vertical-nav__link--disabled--Color:   var(--pf-global--Color--disabled);
+
+  // Vertical nav link background colors
+  --pf-c-vertical-nav__link--BackgroundColor:           var(--pf-global--BackgroundColor--light-100);
+  --pf-c-vertical-nav__link--hover--BackgroundColor:    var(--pf-global--BackgroundColor--light-200);
+  --pf-c-vertical-nav__link--active--BackgroundColor:   var(--pf-global--BackgroundColor--light-200);
+  --pf-c-vertical-nav__link--focus--BackgroundColor:    var(--pf-global--BackgroundColor--light-200);
+  --pf-c-vertical-nav__link--disabled--BackgroundColor: var(--pf-global--BackgroundColor--disabled);
+
+  // Vertical nav link icon
+  --pf-c-vertical-nav__link__icon--MarginRight: var(--pf-global--spacer--xl);
+
+  // Vertical nav link arrow
+  --pf-c-vertical-nav__link__arrow--Top: calc(var(--pf-c-vertical-nav__link--PaddingTop) - .4rem);
+  --pf-c-vertical-nav__link__arrow--Right: var(--pf-global--spacer--xl);
+
+  /* stylelint-enable */
+
+  // List
+  color: var(--pf-c-vertical-nav--Color);
+  background-color: var(--pf-c-vertical-nav--BackgroundColor);
+
+  // List item
+  &__item {
+  }
+
+  // Link
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: baseline;
+    padding-top: var(--pf-c-vertical-nav__link--PaddingTop);
+    padding-right: var(--pf-c-vertical-nav__link--PaddingRight);
+    padding-bottom: var(--pf-c-vertical-nav__link--PaddingBottom);
+    padding-left: var(--pf-c-vertical-nav__link--PaddingLeft);
+    margin-top: var(--pf-c-vertical-nav__link--MarginTop);
```

Comment:
Do we need to change the margin? 
the base `[class^=pf-]` will make it `0` by default

Reply:
No, we don't. This is an assumption based on https://github.com/patternfly/patternfly-next/blob/dbf83c1f88b1097ff560c9e264a9aca7a0e411cb/patternfly_old/pf-c-vertical-nav/pf-c-vertical-nav.scss#L40-L41.

Reply:
removing margin vals